app/models/employee.rb

```diff
@@ -1,6 +1,6 @@
 class Employee < ApplicationRecord
-  has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver_id', inverse_of: :giver, dependent: :destroy
-  has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver_id', inverse_of: :receiver, dependent: :destroy
+  has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver', inverse_of: :giver, dependent: :destroy
```

Comment:
why this has been changed? what does the bullet have to do with it?

Reply:
I am not exactly sure, but i had been struggling with the proper working of bullet gem. Previously when i had specified foreign keys with the ending '_id', the bullet gem didn't alert me in the application on the kudos index page that in the kudos_controller i should add 'includes(:receiver, :giver)'. Therefore, after huge war :D, i found the solution in the removing the '_id' ending from foreig keys. This helps because when i delete 'includes(:receiver, :giver)' from kudos controller i always get the pop-up on the kudos index page from bullet that i should add this phraze. This solution also guarrantee possibility of removing kudos unlike to completely removing foreign keys names from models.